docs/api/sdk/classes/Validator.md

```diff
@@ -1,260 +1,301 @@
----
-id: "Validator"
-title: "Class: Validator"
-sidebar_label: "Validator"
-sidebar_position: 0
-custom_edit_url: null
----
+# Class: Validator
 
 Validator provides direct access to remote Validator REST APIs.
 
 ## Constructors
 
-### constructor
+### new Validator()
 
-• **new Validator**(`config?`): [`Validator`](Validator.md)
+> **new Validator**(`config`): [`Validator`](Validator.md)
 
 Create a Validator instance with the specified connection configuration.
 
 #### Parameters
 
-| Name | Type | Description |
-| :------ | :------ | :------ |
-| `config` | `Partial`\<[`ReadConfig`](../interfaces/helpers.ReadConfig.md)\> | The connection configuration. This must include a baseUrl string. If passing the config from a pre-existing Database instance, it must have a non-null baseUrl key defined. |
+• **config**: `Partial` \<[`ReadConfig`](../namespaces/helpers/interfaces/ReadConfig.md)\>= `{}`
+
+The connection configuration. This must include a baseUrl
+string. If passing the config from a pre-existing Database instance, it
+must have a non-null baseUrl key defined.
 
 #### Returns
 
 [`Validator`](Validator.md)
 
-#### Defined in
+#### Source
 
 @tableland/sdk/src/validator/index.ts:53
 
 ## Properties
 
 ### config
 
-• `Readonly` **config**: [`ReadConfig`](../interfaces/helpers.ReadConfig.md)
+> `readonly` **config**: [`ReadConfig`](../namespaces/helpers/interfaces/ReadConfig.md)
 
-#### Defined in
+#### Source
 
 @tableland/sdk/src/validator/index.ts:46
 
 ## Methods
 
-### getTableById
+### getTableById()
 
-▸ **getTableById**(`params`, `signal?`): `Promise`\<[`Table`](../interfaces/Table.md)\>
+> **getTableById**(`params`, `signal`?): `Promise` \<[`Table`](../interfaces/Table.md)\>
 
 Get table information
 
 #### Parameters
 
-| Name | Type | Description |
-| :------ | :------ | :------ |
-| `params` | `Object` | - |
-| `params.chainId` | `number` | **`Description`** The parent chain to target **`Example`** ```ts 80001 ``` |
-| `params.tableId` | `string` | **`Description`** Table identifier **`Example`** ```ts 1 ``` |
-| `signal?` | [`Signal`](../interfaces/helpers.Signal.md) | - |
+• **params**
+
+• **params.chainId**: `number`
+
+**Description**
+The parent chain to target
+
+**Example**
+```ts
+80001
```

Comment:
Should we change this to 80002?

Reply:
hm, that was autogenerated from the SDK v7.0.0...which is weird because [there's no reference of `80001`](https://github.com/search?q=repo%3Atablelandnetwork%2Ftableland-js+80001&type=code) in any of that code. I've manually changed that, but i wonder where it's coming from?

Reply:
interesting... Looks like the sdk's `npm run build:api` script uses the yaml file located at `https://raw.githubusercontent.com/tablelandnetwork/docs/main/specs/validator/tableland-openapi-spec.yaml` to generate the doc.  Since that url is the `main` branch, the doc won't get generated correctly until after this pr.

a "circular doc generation" issue 🙃

Reply:
here's the actual snippet, which uses 80002:
https://github.com/tablelandnetwork/tableland-js/blob/main/packages/sdk/src/validator/client/validator.ts#L318

but, if i look at my node_modules file, the 80001 chain ID is being used, so the 80002 change must've not been published? anyways, nbd for now because my manual fix will work, and whenever we publish a new SDK version, it'll switch things to 80002